README.md

```diff
@@ -60,6 +63,11 @@ A limited number of properties can be set in the configuration, including the ti
 xnat (DCAT) catalog and the publisher of the catalog. We are still working on adding more default
 values for certain DCAT properties to the configuration.
 
+There is limited support for using environment variables. For setting the XNAT server, variables
+`XNAT_HOST` and `XNATPY_HOST` can be used. Authentication can be provided in `XNAT_USER` and
```

Comment:
> `XNAT_HOST` and `XNATPY_HOST`

"and" vs "or"?

Reply:
You can use both, but XNATPY_HOST takes preference. I updated the readme to clarify this.